client/lib/gsuite/gsuite-product-type.js

```diff
@@ -26,7 +12,12 @@ export function getProductSlug( productType ) {
  * @see emailManagementAddGSuiteUsers() in client/my-sites/email/paths.js
  */
 export function getProductType( productSlug ) {
```

Comment:
It may be worth converting this file to TS, especially given that we only have one function in it. It may also be worth renaming the file to `get-product-type` and exporting that as the default function, but that's a slightly bigger refactor.

Reply:
I would rather handle any conversion in another pull request to ease the review of this one.